requirements_test_all.txt

```diff
@@ -1521,6 +1521,9 @@ rpi-bad-power==0.1.0
 # homeassistant.components.rtsp_to_webrtc
 rtsp-to-webrtc==0.5.1
 
+# homeassistant.components.ruuvitag_ble
+ruuvitag-ble==0.1.0rc1
```

Comment:
Usually we don't accept beta or rc libs?

Can you re-upload this without the `rc1` ?

Reply:
Yep, sure can. Will happen tomorrow 👍

Reply:
Updated to 0.1.1 (non-pre version).